ccm_web/client/src/components/GradebookUploadConfirmationTable.tsx

```diff
@@ -5,6 +5,7 @@ interface StudentGrade {
   rowNumber: number
   uniqname: string
   grade: string
+  overrideGrade: string | undefined

```

Comment:
I feel that when a csv file pass all the validation and if it doesn't have a override grade to that file, I don't feel we should be showing override column in the final table. Thoughts???
![Screen Shot 2021-06-07 at 11 23 05 AM](https://user-images.githubusercontent.com/8579775/121043860-009f1180-c783-11eb-9ba5-6b9d5adbc73d.png)

Reply:
That makes sense.